### PR TITLE
docs: update about portal hosting

### PR DESCRIPTION
**Pull request by Tzal3x**
- The server portal is now hosted on `walrus.site` 
- The service-worker portal is no longer being hosted, but its' code is still available for local hosting.

### CODE REVIEW

**Comment by mlegner** [comment]
@Tzal3x Is the `blob.store` domain going to remain active in the future?

**Comment by Tzal3x** [comment]
> @Tzal3x Is the `blob.store` domain going to remain active in the future?

For the near future we can point it to the server portal so it can be a synonym of walrus.site. But gradually we should start deprecating it.

**Comment by mlegner** [comment]
> > @Tzal3x Is the `blob.store` domain going to remain active in the future?
> 
> For the near future we can point it to the server portal so it can be a synonym of walrus.site. But gradually we should start deprecating it.

Sounds good. Can you mention this in the docs? Something like

> `blob.store` is currently an alias for `walrus.site` but will be removed in the future.

File: docs/walrus-sites/portal.md [hunk 1]
```diff
@@ -8,20 +8,31 @@ portals:
...
-<https://walrus.site>, respectively.
+Currently, only a server-side portal is served at <https://walrus.site>.
+
+```admonish note title="Hosting of the service worker"
+In the past, the service-worker based portal was hosted at <https://walrus.site>
+and the server-portal at <https://blob.store>. This is no longer the case.
+
+The service-worker portal is no longer hosted, but you can run it locally
+because its' code is still available in the `walrus-sites` repository.
+For more information, see [running the portal locally](#running-the-portal-locally).
```
Comment (giac-mysten) [inline docs/walrus-sites/portal.md:19]:
```suggestion
The service-worker portal is no longer hosted, but you can still run it locally. 
Its code is available in the `walrus-sites` repository.
For more information, see [running the portal locally](#running-the-portal-locally).
```

**Review by giac-mysten — APPROVED** [review]
LGTM, please fix the line length!

**Pull request merged**